### PR TITLE
feat: addition of create service account

### PR DESCRIPTION
**Pull request by dimakis**

### CODE REVIEW

File: plugins/module/create_service_account.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        type: str
+    description:
+        description: Description of the service account
+        required: true
+        type: str
+ 
+# Specify this value according to your collection
+# in format of namespace.collection.doc_fragment_name
+extends_documentation_fragment:
+    - redhat.rhoask.my_doc_fragment_name
```
Comment (wtrocki) [inline plugins/module/create_service_account.py:32]: ?

File: plugins/module/create_service_account.py [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+        # Create an instance of the API class
+        api_instance = service_accounts_api.ServiceAccountsApi(api_client)
+        try:
+            service_account_create_request_data = ServiceAccountCreateRequestData(
+                name=module.params['name'],
+                description=module.params['description'],
+            ) 
+            api_response = api_instance.create_service_account(service_account_create_request_data)
+            
+            print('type api_response', api_response)
```
Comment (wtrocki) [inline plugins/module/create_service_account.py:130]: We should not print those values!!

Comment (wtrocki) [reply plugins/module/create_service_account.py:130]: Specifically  api_response['secret'] is `secret` and should not be in output log

Comment (wtrocki) [reply plugins/module/create_service_account.py:130]: See https://www.redhat.com/sysadmin/ansible-playbooks-secrets

Comment (dimakis) [reply plugins/module/create_service_account.py:130]: you're right. damn must have left it in from debugging.

File: plugins/module/create_service_account.py (hunk 1 above)
Comment (dimakis) [reply plugins/module/create_service_account.py:32]: I don't have doc fragments in there yet. I'm going down the route of autodoc gen as per your suggestion, I just need to perfect it, complete docs should be up in a PR soon

**Review by wtrocki — APPROVED** [review]

File: plugins/module/create_service_account.py (hunk 2 above)
Comment (dimakis) [reply plugins/module/create_service_account.py:130]: @wtrocki so regardless, the output of the playbook run is returned and printed to screen, to use the vars `client_id` and `client_secret` we must return them. 
The alternative as per the Ansible docs is to run the play book with the `no_log` param set as per https://docs.ansible.com/ansible/latest/reference_appendices/faq.html#how-do-i-keep-secret-data-in-my-playbook. 
this censors the output but allows the value to be returned. I'm after looking but I really do not see another way of doing this

Comment (dimakis) [reply plugins/module/create_service_account.py:130]: I don't like it but I can't find another way to do it

**Pull request merged**